bitrise.yml

```diff
@@ -51,7 +51,8 @@ workflows:
         inputs:
         - destination: platform=iOS Simulator,name=iPhone 14 Plus,OS=16.4
         - xcodebuild_options: '"COMPILER_INDEX_STORE_ENABLE=NO"
-            "CODE_SIGN_IDENTITY=" "CODE_SIGNING_REQUIRED=NO" "CODE_SIGNING_ALLOWED=NO"'
+            "CODE_SIGN_IDENTITY=" "CODE_SIGNING_REQUIRED=NO" "CODE_SIGNING_ALLOWED=NO" 
+            "-parallel-testing-enabled" "YES" "-parallel-testing-worker-count" "2"'
```

Comment:
This command line is failing on Bitrise. Not sure if the correct format is: 
"-parallel-testing-enabled=YES" "-parallel-testing-worker-count=2" instead

Reply:
I really hope we can enable parallel testing again! is that more stable on Xcode 14.3? 🤞

Reply:
What about the UI Tests?

Reply:
Let me disable parallel testing in unit tests. I can reproduce that the unit tests fail intermittent on my local machine as well. A different set of tests fail in different test runs. :(

Reply:
@clarmso the command line is wrong to enable parallel testing as it is in the PR, see the bitrise log above... it does not even start the tests on bitrise

Reply:
The unit tests started in parallel. `-parallel-testing-enabled` and `-parallel-testing-worker-count` are options from `xcodebuild` and should have syntax `-<option> <arg>`. The other existing arguments are setting environment variables.
https://app.bitrise.io/build/4ab8d625-5a6b-44ba-8bc1-b679584967d4?tab=log

Reply:
I may have looked at an old commit where there was this failure:
https://app.bitrise.io/build/36362eaa-f4f0-41ad-b280-e3ebbc570f39
`rovided xcodebuild options ("COMPILER_INDEX_STORE_ENABLE=NO" "CODE_SIGN_IDENTITY=" "CODE_SIGNING_REQUIRED=NO" "CODE_SIGNING_ALLOWED=NO" "-parallel-testing-enabled" "YES" -parallel-testing-worker-count" "2") are not valid CLI parameters: Unterminated double-quoted string (exit code: 1)`

If that's working then we are good with this :) let's see how this works for the UI tests

Reply:
I run the unit tests on my end using the following command:
```
xcodebuild test -scheme Fennec -configuration Fennec -destination "platform=iOS Simulator,name=iPhone 14,OS=16.4" -testPlan UnitTest -parallel-testing-enabled YES -parallel-testing-worker-count 2
```

Reply:
I've a workflow that got stuck for 10 minutes. Let's do parallel testing another time.
https://app.bitrise.io/build/effcbb50-d4aa-48f4-906f-4e4249bdfeb5